iotiletest/iotile/mock/devices/realtime_test_device.py

```diff
@@ -27,29 +27,30 @@ class RealtimeTestDevice(VirtualIOTileDevice):
         args (dict): Any arguments that you want to pass to create this device.
             Supported args are:
 
-                iotile_id (int)
-                    The UUID used for this device.  If no UUID is specified, the default value of 1 is used.
-
-                streams (dict):
-                    A map of strings with hex numbers to tuples of (interval, value)
-                    where interval is a float that expresses how often the stream should stream
-                    in seconds and value is an integer that is sent as the value every interval
-                    as a realtime reading (IndividualReadingReport).  The stream id is the key of the
-                    streams dict which should be a string encoding of a hex number including
-                    the prefix 0x so that it can be parsed with int(key, 0).
-
-                broadcast (dict):
-                    A map of strings with hex numbers to tuples of (interval, value)
-                    where interval is a float that expresses how often the stream should stream
-                    in seconds and value is an integer that is sent as the value every interval
-                    as a broadcast reading (BroadcastReport).  The stream id is the key of the
-                    streams dict which should be a string encoding of a hex number including
-                    the prefix 0x so that it can be parsed with int(key, 0).
-
-                trace (list):
-                    A list of tuples which are (float, string) lists
-                    that will trace the fixed string every fixed interval given
-                    by the first float argument in seconds.
+                iotile_id (int) The UUID used for this device.  If no UUID is
+                    specified, the default value of 1 is used.
+
+                streams (dict): A map of strings with hex numbers to tuples of
+                    (interval, value) where interval is a float that expresses
+                    how often the stream should stream in seconds and value is
+                    an integer that is sent as the value every interval as a
+                    realtime reading (IndividualReadingReport).  The stream id
+                    is the key of the streams dict which should be a string
+                    encoding of a hex number including the prefix 0x so that
+                    it can be parsed with int(key, 0).
+
+                broadcast (dict): A map of strings with hex numbers to tuples
```

Comment:
If you specify multiple broadcasts, I seem to recall the behavior where you would only see one value at a time being broadcast, with the most recent one overwriting whatever was there before. It might help to clarify that behavior here in case someone comes across it?

Reply:
Added additional comments in the docstring about this